packages/core/test/client.test.ts

```diff
@@ -72,6 +72,46 @@ describe('@bugsnag/core/client', () => {
       expect(client).toBeTruthy()
       expect(logger.warn.mock.calls.length).toBe(0)
     })
+
+    it('prevents sendSession() with invalid endpoint config', done => {
+      // @ts-expect-error
+      const client = new Client({ apiKey: 'API_KEY', endpoints: { notify: '/notify' } })
+
+      client._setDelivery(client => ({
+        sendSession: () => { done('sendSession() should not be called') },
+        sendEvent: () => { }
+      }))
+
+      client.notify(new Error('111'), () => {}, (err, event) => {
+        expect(err).toBe(null)
+        expect(event).toBeTruthy()
+        expect(event.errors[0].errorMessage).toBe('111')
+        done()
+      })
+
+      // give the event loop a tick to see if the event gets sent
+      process.nextTick(() => done())
```

Comment:
not sure i understand this. won't this just cause the test to pass regardless of what assertions above fail?

Reply:
I'm not sure either - I copied it from a similar test but I think it needs to go altogether.